trade-finance-manager-ui/server/controllers/utilisation-reports/record-corrections/create-record-correction-request/index.test.ts

```diff
@@ -1,13 +1,23 @@
 import httpMocks from 'node-mocks-http';
+import { RECORD_CORRECTION_REASON } from '@ukef/dtfs2-common';
 import { aTfmSessionUser } from '../../../../../test-helpers';
-import { createRecordCorrectionRequest, CreateRecordCorrectionRequestRequest } from '.';
-import { CreateRecordCorrectionRequestViewModel } from '../../../../types/view-models';
+import {
+  EMPTY_CREATE_RECORD_CORRECTION_REQUEST_ERRORS_VIEW_MODEL,
+  getCreateRecordCorrectionRequest,
+  GetCreateRecordCorrectionRequestRequest,
+  PostCreateRecordCorrectionRequestRequest,
+  postCreateRecordCorrectionRequest,
+} from '.';
+import { CreateRecordCorrectionRequestErrorsViewModel, CreateRecordCorrectionRequestViewModel } from '../../../../types/view-models';
+import { PRIMARY_NAVIGATION_KEYS } from '../../../../constants';
+import { CreateRecordCorrectionRequestFormRequestBody } from './form-helpers';
+import { validateCreateRecordCorrectionRequestFormValues } from './validate-form-values';
 import api from '../../../../api';
 import { GetFeeRecordResponseBody } from '../../../../api-response-types';
-import { PRIMARY_NAVIGATION_KEYS } from '../../../../constants';
 import { getLinkToPremiumPaymentsTab } from '../../helpers';
 
 jest.mock('../../../../api');
+jest.mock('./validate-form-values');
```

Comment:
Instead of mocking this, we can just assert that the controller renders the nunjucks template with this:

```
errors: validateCreateRecordCorrectionRequestFormValues(formValues),
```

Reply:
Thanks 😃, simplified to this